jobs/check-assessment-exercise-ids.groovy

```diff
@@ -0,0 +1,41 @@
+// Loads all the assessment configs from datastore and checks to make sure that
+// their ExerciseIDs are valid (i.e., that there are published exercises with
+// those IDs).
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+
+
+new Setup(steps).apply();
```

Comment:
You want to run this as a cronjob, right?  So you need to add a cron step here.  Though maybe you're planning to do that later once you confirm this works?  I'd probably do it from the beginning, tbh.  It's pretty straightforward what you're doing and all the work is in the go binary, so this won't take a lot of iterating.

Reply:
Ah okay, I thought I was supposed to configure that in Jenkins! Will add it now